tests/shapes.nim

```diff
@@ -1,36 +1,40 @@
 import std/[unittest, math, options]
-import PhotoNim/[shapes, geometry, camera]
+import PhotoNim
+
+proc areClose(a, b: HitRecord): bool {.inline.} = 
```

Comment:
Ok i like that you are defining areClose procedure for HitRecord type here because we just don't need to have it elsewhere but it's useful only for debugging procedures